src/components/Message/styles/Message.css.js

```diff
@@ -60,6 +60,15 @@ const css = `
     margin-bottom: 0;
     min-width: initial;
   }
+
+  .c-MessageChatBlock:only-child .c-MessageBubble.emoji-only {
+    padding-top: 0 !important;
+    padding-bottom: 0 !important;
+  }
+
+  .c-MessageChatBlock:last-child .c-MessageBubble.emoji-only {
```

Comment:
This is an override for when a message block ends with a message that includes an emoji.
<img width="535" alt="Screen Shot 2020-03-03 at 15 41 23" src="https://user-images.githubusercontent.com/7111256/75823173-e1872800-5d66-11ea-98b9-bcb84d5953ba.png">